react/components/MiniCartContent.js

```diff
@@ -68,43 +74,26 @@ class MiniCartContent extends Component {
     this.sumItemsPrice(items) - totalPrice
 
   handleItemRemoval = async id => {
-    this.updateItemLoad(id, true)
-
     const {
-      data: orderFormContext,
-      data: { orderForm, updateAndRefetchOrderForm },
-      intl,
+      data: { orderForm },
+      updateItems,
     } = this.props
     const itemPayload = orderForm.items.find(item => item.id === id)
     const index = orderForm.items.indexOf(itemPayload)
-    const updatedItem = [itemPayload].map(item => {
-      return {
-        id: parseInt(item.id),
-        index: index,
-        quantity: 0,
-        seller: 1,
-      }
-    })
+    const updatedItem = [itemPayload].map(({ id, ...rest }) => ({
+      id,
+      ...rest,
+      index,
+      quantity: 0,
+      seller: 1,
+    }))
 
     try {
-      await updateAndRefetchOrderForm({
-        variables: {
-          orderFormId: orderForm.orderFormId,
-          items: updatedItem,
-        },
-      })
+      await updateItems(updatedItem)
     } catch (error) {
-      // TODO improve the way this error is presented.
-      orderFormContext.updateToastMessage({
-        isSuccess: false,
-        text: intl.formatMessage({ id: 'minicart.error-removal' }),
-      })
-
-      window.setTimeout(() => {
-        orderFormContext.updateToastMessage({ isSuccess: null, text: null })
-      }, TOAST_TIMEOUT)
+      // TODO: Toast error message
```

Comment:
Do you expect this to be resolved in this PR?

Reply:
It can be resolved later. Other apps need to be changed to do this.